toc.yml

```diff
@@ -1,5 +1,6 @@
 - Banking doctypes: ./docs/io.cozy.bank.md
 - Bills: ./docs/io.cozy.bills.md
+- Salaries: ./docs/io.cozy.salaries.md
```

Comment:
I prefer the term payslip : you can have a payslip without being a salary man.

Reply:
Then let's go for `io.cozy.payslips` (it was my first guess) but how to you call the attribute for the file (`payslip` at the moment) ?

Reply:
file ?

Reply:
I opted for a more generic `document` $relationship attribute